### ISSUE TITLE
Unrecognized attribute: min for operator Clip

### ISSUE THREAD

**Issue by killTheHostage** [pytorch/pytorch#39053]
Hello, guys. Recently when I was using pytorch to export the WS-DAN onnx model, pytorch reported an error Unrecognized attribute: min for operator Clip
According to [THIS ISSUE](https://github.com/onnx/onnx/issues/2528#issuecomment-614892761), it seems that there is a problem with pytorch's implementation
I have no idea if it's a bug or something else. Please help me to solve this problem
THANKS A LOT.

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by killTheHostage** [comment]
- P.S I'm using the latest stable version(1.5.0)

**Comment by killTheHostage** [comment]
Traceback are as follow:
Traceback (most recent call last):
  File "export_onnx.py", line 22, in <module>
    output_names=output_names
  File "/home/xxx/anaconda3/lib/python3.7/site-packages/torch/onnx/__init__.py", line 168, in export
    custom_opsets, enable_onnx_checker, use_external_data_format)
  File "/home/xxx/anaconda3/lib/python3.7/site-packages/torch/onnx/utils.py", line 69, in export
    use_external_data_format=use_external_data_format)
  File "/home/xxx/anaconda3/lib/python3.7/site-packages/torch/onnx/utils.py", line 515, in _export
    _check_onnx_proto(proto)
RuntimeError: Unrecognized attribute: min for operator Clip

Source code are as follow:
ws_dan = WSDAN(num_classes=2, M=8, net="xception")
    ws_dan_checkpoint = torch.load("/home/xxx/wsdan-onnx/output/dfdc-wsdan-xception_1/ckpt_30.pth")
    ws_dan.load_state_dict(ws_dan_checkpoint["state_dict"])
    ws_dan.eval()
    dummy_input = torch.randn(32, 3, 320, 320)
    input_names = ["input"]
    output_names = ["output"]
    opset_version = 12
    onnx_output_path = "./ws-dan_xception_b32_320.onnx"
    torch.onnx.export(
        ws_dan,
        dummy_input,
        onnx_output_path,
        opset_version=opset_version,
        input_names=input_names,
        output_names=output_names
    )

**Comment by killTheHostage** [comment]
By the way, I have no choice but set `opset_version = 12`, operator "einsum" also used in this model

**Comment by spandantiwari** [comment]
Can you share the repro code, i.e. the model code and your export code? Also, note that in PyTorch 1.5 only Opset 11 is supported. Opset 12 is not supported and is partial at best.

**Comment by spandantiwari** [comment]
@killTheHostage - I am not able to repro this issue in Opset 11 myself. In opset 11 Clip operator should be exported with min and max as inputs and not attributes as seen [here](https://github.com/pytorch/pytorch/blob/d1352192e21001521035dacc0eda574b58224198/torch/onnx/symbolic_opset11.py#L45) and [here](https://github.com/pytorch/pytorch/blob/d1352192e21001521035dacc0eda574b58224198/torch/onnx/symbolic_opset11.py#L30).
As I asked earlier - if you can you share the model code (it will be great if you can create a smaller repro with skeleton model topology (doesn't have to be the final trained model)) and your exact export code, we will take a look.

**Comment by spandantiwari** [comment]
@killTheHostage - could you please share a repro.

**Comment by spandantiwari** [comment]
@killTheHostage - I will close this issue for now. Please feel free to reopen if you have a repro for this issue and we will look into this. Thanks.

**Comment by boy2000-007man** [comment]
@spandantiwari 
I can repro this issue with `torch.nn.functional.normalize`
```python3
import torch, onnxruntime
print(torch.__version__, onnxruntime.__version__)
class M(torch.nn.Module):
    def forward(self, x):
        return torch.nn.functional.normalize(x)
torch.onnx.export(M(), torch.rand(3,4), "debug.onnx", opset_version=11)
onnxruntime.InferenceSession("debug.onnx")
```
outputs
```python3
1.4.0 1.4.0
---------------------------------------------------------------------------
InvalidGraph                              Traceback (most recent call last)
<ipython-input-2-53f924e284ac> in <module>()
      5         return torch.nn.functional.normalize(x)
      6 torch.onnx.export(M(), torch.rand(3,4), "debug.onnx", opset_version=11)
----> 7 onnxruntime.InferenceSession("debug.onnx")

/home/.local/lib/python3.6/site-packages/onnxruntime/capi/session.py in __init__(self, path_or_bytes, sess_options, providers)
    156         self._path_or_bytes = path_or_bytes
    157         self._sess_options = sess_options
--> 158         self._load_model(providers or [])
    159         self._enable_fallback = True
    160         Session.__init__(self, self._sess)

/home/.local/lib/python3.6/site-packages/onnxruntime/capi/session.py in _load_model(self, providers)
    175             raise TypeError("Unable to load from type '{0}'".format(type(self._path_or_bytes)))
    176 
--> 177         self._sess.load_model(providers)
    178 
    179         self._sess_options = self._sess.session_options

InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : This is an invalid model. Error in Node: : Unrecognized attribute: min for operator Clip
```

**Comment by killTheHostage** [comment]
> @spandantiwari
> I can repro this issue with `torch.nn.functional.normalize`
> 
> ```python
> import torch, onnxruntime
> print(torch.__version__, onnxruntime.__version__)
> class M(torch.nn.Module):
>     def forward(self, x):
>         return torch.nn.functional.normalize(x)
> torch.onnx.export(M(), torch.rand(3,4), "debug.onnx", opset_version=11)
> onnxruntime.InferenceSession("debug.onnx")
> ```
> 
> outputs
> 
> ```python
> 1.4.0 1.4.0
> ---------------------------------------------------------------------------
> InvalidGraph                              Traceback (most recent call last)
> <ipython-input-2-53f924e284ac> in <module>()
>       5         return torch.nn.functional.normalize(x)
>       6 torch.onnx.export(M(), torch.rand(3,4), "debug.onnx", opset_version=11)
> ----> 7 onnxruntime.InferenceSession("debug.onnx")
> 
> /home/.local/lib/python3.6/site-packages/onnxruntime/capi/session.py in __init__(self, path_or_bytes, sess_options, providers)
>     156         self._path_or_bytes = path_or_bytes
>     157         self._sess_options = sess_options
> --> 158         self._load_model(providers or [])
>     159         self._enable_fallback = True
>     160         Session.__init__(self, self._sess)
> 
> /home/.local/lib/python3.6/site-packages/onnxruntime/capi/session.py in _load_model(self, providers)
>     175             raise TypeError("Unable to load from type '{0}'".format(type(self._path_or_bytes)))
>     176 
> --> 177         self._sess.load_model(providers)
>     178 
>     179         self._sess_options = self._sess.session_options
> 
> InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : This is an invalid model. Error in Node: : Unrecognized attribute: min for operator Clip
> ```

THANK YOU VERY MUCH
Because I have given up exporting the ONNX model of WS-DAN and switched to other networks without problems, I did not continue to follow up on this issue. THANK YOU FOR YOUR REPRO AGAIN. I almost forgot about this 😂

**Comment by boy2000-007man** [comment]
seems fixed and can't repro with `pytorch 1.6`
